torch/csrc/jit/mobile/import.h

```diff
@@ -72,6 +72,20 @@ void _load_extra_only_for_mobile(
     c10::optional<at::Device> device,
     ExtraFilesMap& extra_files);
 
+// Currently used by both mobile/import.cpp and model_compatibility.cpp.
+// Should be removed after model_compatibility.cpp start using simplified
+// version type_resolver and obj_loader.
+at::TypePtr resolveTypeNameGeneral(
```

Comment:
Those functions are for mobile only. Could we rename them to *Mobile?

Reply:
Address in latest commit.